src/main/java/com/minecolonies/colony/jobs/JobBuilder.java

```diff
@@ -119,14 +120,11 @@ public WorkOrderBuild getWorkOrder()
     }
 
     /**
-     * Set a Work Order for this Job
+     * Set a Work Order for this Job.
      *
      * @param order Work Order to associate with this job, or null
      */
-    public void setWorkOrder(WorkOrderBuild order)
-    {
-        workOrderId = (order != null) ? order.getID() : 0;
-    }
+    public void setWorkOrder(WorkOrderBuild order) { workOrderId = (order == null) ? 0 : order.getID(); }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)